### DISCUSSION TITLE
Include Python request examples in OpenAPI

### DISCUSSION THREAD

**Discussion by arredond** [GitbookIO/community#605] [4. Open API] [answered]
Python is the second most popular programming language in the world, extensively used for scripting and interacting with APIs. We write most of our code examples in Python but the auto-generated OpenAPI request examples only including curl and JS, which creates a weird mismatch feeling.

As a bonus, https://curlconverter.com/ tends to convert from curl to Python (+ the `requests` library) extremely well.

**Comment by aikoii** [comment, marked as answer]
Hey @arredond! We already support Python examples in the OpenAPI block 👍
